data/food/soup.json

```diff
@@ -0,0 +1,54 @@
+{
+    "templates": [
+        "hands {user} a {soup} soup."
+    ],
+    "parts": {
+        "soup": [
+" Bowl of Ajiaco ",
+" Bowl of Avgolemono ",
+" Bowl of Borscht ",
+" Bowl of Beef noodle soup",
+" Bowl of Beer soup",
+" Bowl of Birds nest soup",
+" Bowl of Chicken noodle soup",
+" Bowl of Caldo verde ",
+" Bowl of Cazuela",
+" Bowl of Chicken Noodle Soup",
+" Bowl of Cock-a-leekie",
+" Bowl of Fufu and Egusi soup",
+" Bowl of Gomguk ",
+" Bowl of Goulash soup ",
+" Bowl of Gumbo",
+" Bowl of  and sour soup ",
+" Bowl of Kharcho",
+" Bowl of Kimchi Guk ",
+" Bowl of Lagman",
+" Bowl of Leek soup",
+" Bowl of Lentil soup",
+" Bowl of Matzah ball soup",
```

Comment:
These options would result in `hands <user> a Bowl of Matzah ball soup soup` due to the fact that the template itself adds the word 'soup'